pages/common/nmap.md

```diff
@@ -32,6 +32,6 @@
 
 `nmap --script "http-*" {{ip_or_host1,ip_or_host2...}} -p 80,443`
 
-- Attempt IDS/IPS detection by using an extremely slow scan (`-T0`), decoy source addresses (`-D`), [f]ragmented packets, random data and other methods:
+- Attempt evading IDS/IPS detection by using an extremely slow scan (`-T0`), decoy source addresses (`-D`), [f]ragmented packets, random data and other methods:
 
 `sudo nmap -T0 -D {{decoy_ip1,decoy_ip2...}} --source-port {{53}} -f --data-length {{16}} -Pn {{ip_or_host}}`
```

Comment:
All placeholders are missing a comma before the ellipsis.
```suggestion
`sudo nmap -T0 -D {{decoy_ip1,decoy_ip2,...}} --source-port {{53}} -f --data-length {{16}} -Pn {{ip_or_host}}`
```

Reply:
> All placeholders are missing a comma before the ellipsis.

Thanks, fixed.